### DISCUSSION TITLE
20241205-spamfilter/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#60] [Comments]
# 20241205-spamfilter/

AI/ML 전문가 없이 자동화된 스팸 필터링 도입과 개선기에 대해 공유합니다.

https://tech.inflab.com/20241205-spamfilter/

**Comment by iamb0ttle** [comment]
머신러닝 모델을 자동화하고 슬랙 봇으로 스팸메시들을 트래킹하시는게 신기하네요 ㄷㄷ 저도 기회가 된다면 프로젝트에 적용해보고 싶은 마음이 드는 글이였습니다 ㅎㅎ
